### PR TITLE
Terraform

### PR DESCRIPTION
**Pull request by trumanshuck**
@go-between/folks 

Adds a staging environment and does this-and-that to work more nicely in terraform.  Corresponds to this terraform PR:  https://github.com/go-between/musicbox-terraform/pull/3.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 require "active_record/railtie"
 # require 'active_storage/engine'
 require "action_controller/railtie"
-# require 'action_mailer/railtie'
+require "action_mailer/railtie"
```
Comment (trumanshuck) [inline config/application.rb:12]: Even when we remove all of the email stuff from devise, it _still_ gets angry if we don't have action mailer.  Will have to look more at this later.

Comment (sisk) [reply config/application.rb:12]: I'm curious to know what was happening. I removed this and was able to run full specs, server and console. is there some sort of deployment/bundling thing?

Comment (sisk) [reply config/application.rb:12]: never mind. I saw [this](https://github.com/go-between/musicbox-api/pull/61/commits/f388fc99d26dc6c02803c5500cb8e4a6d00985d3).

File: config/environments/production.rb [hunk 2]
```diff
@@ -43,7 +43,7 @@
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
-  config.log_level = :debug
+  config.log_level = :error
```
Comment (sisk) [inline config/environments/production.rb:46]: I'd probably `info` or `warn` here. wouldn't error clobber the others?

Comment (sisk) [reply config/environments/production.rb:46]: we might even want to make that variable if all of a sudden we wanted to drop prod to debug mode without a full deployment.

**Pull request merged**